third_party/libusb/naclport/src/chrome_usb/chrome-usb-backend.js

```diff
@@ -222,12 +290,15 @@ ChromeUsbBackend.prototype.reportRequestError_ = function(
 
 /**
  * @param {string} debugRepresentation
+ * @param {string} functionName API function that was called
  * @param {!goog.promise.Resolver} promiseResolver
  * @param {!Array} resultArgs
  * @private
  */
 ChromeUsbBackend.prototype.reportRequestSuccess_ = function(
-    debugRepresentation, promiseResolver, resultArgs) {
+    debugRepresentation, functionName, promiseResolver, resultArgs) {
+  this.requestSuccessHooks_.forEach(function(hook) {
+      resultArgs = hook(functionName, resultArgs);});
```

Comment:
nit: -2 spaces of indentation here.
Also normally the "}" closing bracket should go into a new line.